tests/e2e/tests/01-enable-proxy-mode/02-assert.yaml

```
@@ -0,0 +1,7 @@
+apiVersion: elasti.truefoundry.com/v1alpha1
+kind: ElastiService
+metadata:
+  name: target-elastiservice
+  namespace: default
+status:
+  mode: proxy
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)